src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5159,15 +5158,15 @@ public partial class FileStream : System.IO.Stream
         public override int Read(byte[] array, int offset, int count) { throw null; }
         public override int Read(System.Span<byte> buffer) { throw null; }
         public override System.Threading.Tasks.Task<int> ReadAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
-        public override System.Threading.Tasks.ValueTask<int> ReadAsync(System.Memory<byte> buffer, System.Threading.CancellationToken cancellationToken = default) { throw null; }
+        public override System.Threading.Tasks.ValueTask<int> ReadAsync(System.Memory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
It'd be nice if the tool didn't require expanding this, but I realize that's how it's currently implemented.

Reply:
Yep. We should fix it. https://github.com/dotnet/arcade/issues/1341